content/docs/1.7.0/advanced-resources/backing-image/backing-image.md

```diff
@@ -24,6 +24,14 @@ You can prepare a backing image using four different kinds of data sources.
 - It's recommended to provide the expected checksum during backing image creation.
   Otherwise, Longhorn will consider the checksum of the first file as the correct one. Once there is something wrong with the first file preparation, which then leads to an incorrect checksum as the expected value, this backing image is probably unavailable.
 
+#### Scheduling
+- The backing image file will be first prepared and stored on a random node and a random disk. The file then be duplicated to the disks where replicas are.
+- Users can add `nodeSelector` and `diskSelector` to force BackingImage files to be stored on specific node set and disk set to improve space efficiency.
+- Note that if the `nodeSelector` and `diskSelector` settings are not the same between the volume and the backing image, the replicas will not be able to start if they are scheduled to nodes or disks where backing image can not be placed.
```

Comment:
```suggestion
- Longhorn first prepares and stores the backing image file on a random node and disk, and then duplicates the file to the disks that are storing the replicas.
- For improved space efficiency, you can add `nodeSelector` and `diskSelector` to force storing of backing image files on a specific set of nodes and disks.
- The replicas will not start if: 1) the `nodeSelector` and `diskSelector` settings for the volume and backing image are not identical, and 2) the replicas are scheduled on nodes or disks that cannot store the backing image.
```

Reply:
Hi @jillian-maroket 
> - The replicas will not start if: 1) the `nodeSelector` and `diskSelector` settings for the volume and backing image are not identical, and 2) the replicas are scheduled on nodes or disks that cannot store the backing image.

The modification is not correct.
Here is the context
- volume can have multiple replicas
- replicas will be scheduled based on the nodeSelector and diskSelector of the volume
- BackingImage has its own nodeSelector and diskSelector  as well
- if a replica is scheduled to the node where BackingImage can not be scheduled then it won't able to start.

So **it is okay** that **"the `nodeSelector` and `diskSelector` settings for the volume and backing image are not identical"**

Reply:
But the design has changed, I will provide a new design in the doc, please take a look after, thanks